docs/UserGuide.md

```diff
@@ -179,18 +183,79 @@ Let's add for example, `CS2113T` to the timetable
+		THU		                                                                                                                                                                                                
+				                                                                                                                                                                                                
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+				                                                                                                                                                                                                
+		FRI		                                                                                                                                                                                                
+				                                                                                                                                                                                                
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+				                                                                                                                                                                                                
+		SAT		                                                                                                                                                                                                
+				                                                                                                                                                                                                
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+				                                                                                                                                                                                                
+		SUN		                                                                                                                                                                                                
+				                                                                                                                                                                                                
+
+*******************
+Modules taken this semester: 
+
+
+Total MCs taken this semester: 0.0
+
```

Comment:
Not necessary to show an empty timetable or timetable command within Clear Command